.github/workflows/jlab-a11y-test.yml

```diff
@@ -13,16 +13,6 @@ jobs:
       - name: Checkout repository 🛎
         uses: actions/checkout@v3
 
-      - name: Set up Python 🐍
-        uses: actions/setup-python@v4
-        with:
-          python-version: 3.9
-
-      - name: Set up Node 🧶
-        uses: actions/setup-node@v3
-        with:
-          node-version: "14.x"
-
```

Comment:
These are specified in the environment.yml file. Was there a reason to install them separately?